src/renderers/shaders/ShaderChunk/occlusion_fragment.glsl.js

```diff
@@ -0,0 +1,20 @@
+export default /* glsl */`
+#ifdef USE_OCCLUSION
+    if (gl_FragColor.a > 0.0) {
+        int arrayIndex = 0;
+        vec2 depthUv;
+        if (gl_FragCoord.x>=depthWidth) {
+            arrayIndex = 1;
+            depthUv = vec2((gl_FragCoord.x-depthWidth)/depthWidth, gl_FragCoord.y/depthHeight);
+        } else {
+            depthUv = vec2(gl_FragCoord.x/depthWidth, gl_FragCoord.y/depthHeight);
+        }
+        float assetDepthM = gl_FragCoord.z;
+        
+        //float occlusion = Depth_GetOcclusion(depthColor, depthUv, assetDepthM, arrayIndex);
```

Comment:
I think this is possibly a leftover comment? maybe we could remove it

Reply:
I left it here for experimentation. I suspect that we want to tweak the shader but happy to remove it.